docs/whats-new/staging-release-notes.md

```diff
@@ -1,5 +1,55 @@
 # Staging release notes
 
+## Staging release 03 August 2022
+Frontend version: x | Backend version: x  
```

Comment:
will the version numbers be available only on 3rd Aug?

Reply:
yes